front/temporal/labs/activities.ts

```diff
@@ -228,22 +258,32 @@ export async function processGoogleDriveTranscriptActivity(
     return;
   }
 
-  // TODO: Consider using a content fragment to attach the transcript.
   const convRes = await dustAPI.createConversation({
-    title: null,
+    title: transcriptTitle,
     visibility: "unlisted",
     message: {
-      content: transcriptContent + extraInstructions,
+      content: "Transcript: " + transcriptTitle,
```

Comment:
We should mention the assistant here so that the mention is properly rendered in the conversation if visited. We use a markdown directive for mentions:

`:mention[ASSISTANT_NAME]{sId=ASSISTANT_SID}`

The message here should be:

`:mention[ASSISTANT_NAME]{sId=ASSISTANT_SID} Transcript: TRANSCRIPT_TITLE`

I'm not completely convinced by the utility of passing the transcript title as message? Maybe we want to simply mention the assistant no?

Reply:
(the title will be repeated as we render the content fragment which has the title as well no?)